cosmos/precompile/governance/governance.go

```diff
@@ -96,6 +96,7 @@ func (c *Contract) PrecompileMethods() coreprecompile.Methods {
 // SubmitProposal is the method for the `submitProposal` method of the governance precompile contract.
 func (c *Contract) SubmitProposal(
 	ctx context.Context,
+	_ coreprecompile.EVM,
```

Comment:
Why not use core.PrecompileEVM?

Reply:
in eth folder, i just imported `PrecompileEVM` from Geth into the precompile package. `coreprecompile` is meh but had to do that alias because theres a precompile import from cosmos folder